src/main/java/lebron/Ui.java

```diff
@@ -0,0 +1,109 @@
+package lebron;
+
+import lebron.task.Task;
+import lebron.task.TaskList;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a response the bot makes to the user.
+ *
+ * @author Nigel Tan
+ */
+public class Ui {
+    private final String HORIZONTAL_LINE = "    ____________________________________________________________\n";
```

Comment:
Line looks longer than 110 characters. Don't you think It would be ideal to split it into two lines to avoid very long lines?

Reply:
Thanks for spotting this!